compose.yaml

```diff
@@ -10,5 +10,9 @@ services:
       - POSTGRES_DB=${DATABASE_NAME}
     ports:
       - "8010:5432"
+    networks:
```

Comment:
These changes to compose should be their own PR- this docker network technique has technically been approved for the ETL.